app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -6,19 +6,39 @@ body: |
   that would be useful to them. This component would be used to add this personalised content and would
   indicate to the user that this is not normally part of the page, but has been added for them specifically.
 
-  Right now the contents of the component are static, as the MVP of personalised content is only for Start a Business.
-  Since many pages will use this component with the same text, we hard-code it here for now.
+  Parameters:
+
+    - `suggestion_text`: the text to display first, before the link
+    - `suggestion_link_text`: the text of the link
+    - `suggestion_link_text`: the URL of the link
+    - `dismiss_text`: the text of the link used to dismiss the intervention
+
+  The dismiss link points to the same page with the "?hide-intervention" query string. The backend should check for this string in order to render the same page without the intervention shown.
```

Comment:
Have you tested that this will work as expected? I think `?hide-intervention=true` will be appended to the end of the current URL, but does that work correctly where there are already URL parameters or window hashes, or both? e.g.

```
https://www.gov.uk/vehicles-can-drive?step-by-step-nav=e01e924b-9c7c-4c71-8241-66a575c2f61f
https://www.gov.uk/guidance/covid-19-coronavirus-restrictions-what-you-can-and-cannot-do?priority-taxon=774cee22-d896-44c1-a611-e3109cce8eae#understanding-the-risks-of-covid-19
```
```